extensions/.ratignore

```diff
@@ -0,0 +1 @@
+guacamole-auth-radius/**/*
```

Comment:
So does the RADIUS extension not get checked at all? It is excluded, here, and the RAT stuff is also removed from the RADIUS extension's pom.xml file. Does the fact that it's got (L)GPL dependencies mean that we don't do any checking on it?

Reply:
It's checked when it's built by the RAT plugin (it will automatically run when the build is triggered for all projects now, including guacamole-auth-radius).

The issue here is that guacamole-auth-radius is not part of the build unless explicitly included with `-Plgpl-extensions`. The RAT plugin will helpfully exclude modules from the check, but that exclusion won't occur for guacamole-auth-radius if it's not part of the current build. This forces that module to be ignored regardless of whether it's part of the build.